src/main/java/school/hei/patrimoine/visualisation/swing/ihm/google/GoogleSubmitScreen.java

```diff
@@ -138,38 +162,48 @@ protected void done() {
     loadingDialog.setVisible(true);
   }
 
-  private LinkedPatrimoine<NamedString> extractInputData() {
-    List<NamedString> linkDataList = new ArrayList<>();
-    String possessionLink = "";
+  private GoogleLinkList<NamedString> extractInputData() {
```

Comment:
create method extractInputData(List<NamedString>) and use it twice, once on docsField and the second time on drive, the merge results

Reply:
done